src/udev/udev-rules.c

```diff
@@ -2053,21 +2053,20 @@ static int udev_rule_apply_token_to_event(
                 if (token->op == OP_ASSIGN_FINAL)
                         event->name_final = true;
 
+                if (sd_device_get_ifindex(dev, NULL) < 0) {
+                        log_rule_error(dev, rules,
+                                       "Only network interface can be renamed, ignoring NAME=\"%s\"; please fix it.",
```

Comment:
"please fix it" is unclear: fix what and how? Also, the fact that this is printed in red is enough to tell the reader that it's something to fix.

> `"Only network interfaces can be renamed, ignoring NAME=\"%s\"."`